### PR TITLE
678: Add mapping for the grade code. Currently hardcoding the list.

### PR DESCRIPTION
**Pull request by dconey646**

### CODE REVIEW

File: app/context-map/omGradeCode.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (caoimheb) [inline app/context-map/omGradeCode.js]: Rename file to use kebab case?

File: app/context-map/omGradeCode.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    newGradeCode = 'PO'
+  } else if (spoList.includes(omGradeCode)) {
+    newGradeCode = 'SPO'
+  } else if (tpoList.includes(omGradeCode)) {
+    newGradeCode = 'TPO'
+  } else if (psoList.includes(omGradeCode)) {
+    newGradeCode = 'PSO'
+  }
+  return newGradeCode
+}
```
Comment (caoimheb) [inline app/context-map/omGradeCode.js]: All the codes mapped to `DMY` in the spreadsheet will have a blank new code. Is this what we want / expect?

Comment (caoimheb) [inline app/context-map/omGradeCode.js]: Add unit tests?

Comment (dconey646) [inline app/context-map/omGradeCode.js]: I completely missed the unit tests for the current context mapper....

File: app/context-map/omGradeCode.js (hunk 1 above)
Comment (dconey646) [inline app/context-map/omGradeCode.js]: Yep, good spot

File: app/context-map/omGradeCode.js (hunk 2 above)
Comment (dconey646) [inline app/context-map/omGradeCode.js]: This is open for discussion....adding as is at the moment.

File: test/unit/context-map/test-om-grade-code.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    it('returns the correct grade code when an nDelius code is passed in', () => {
+      expect(filterOmGradeCode('NPSD')).to.equal('PO')
+      expect(filterOmGradeCode('CRCC')).to.equal('SPO')
+      expect(filterOmGradeCode('CRCK')).to.equal('TPO')
+      expect(filterOmGradeCode('NPSY')).to.equal('PSO')
+    })
+  })
+  describe('invalid codes', () => {
+    it('return an empty string', () => {
+
```
Comment (caoimheb) [inline test/unit/context-map/test-om-grade-code.js]: Test missing?

**Pull request merged**